game.js

```diff
@@ -68,15 +70,20 @@ class GameRoom extends Room {
     })
   }
 
-  onJoin (client) {
+  onJoin (client, options) {
     console.log('client ', client.sessionId, ' joined')
 
+    const playerName = options?.name
```

Comment:
We'll keep this, but we'll just need to keep in mind that Node 12 won't be supported